docs/_blog/liana-payjoin.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Automatically Refresh Inheritance Time-Locks with Payjoin"
+description: "Liana combines Payjoin and time-locked inheritance to help you keep your Bitcoin private now and safe for those who’ll need it later."
+date: "2025-04-10"
+authors:
+  - Armin Sabouri
+tags:
+  - privacy
+  - integrations
+---
+
+[Liana](https://wizardsardine.com/liana/) is a Bitcoin desktop wallet that enables users to set up time-locked inheritance policies, ensuring their funds can be passed on to loved ones. To prevent unintentional recovery path activation, these time-locked coins require periodic refreshing. Liana’s Payjoin integration enables users to refresh timelocks as they receive payments, boosting privacy and cutting costs.
+
+## Privacy Leak Explained
+
+Inheritance planning is a crucial yet often overlooked part of Bitcoin self-custody. While Bitcoin offers unmatched financial sovereignty, it introduces [unique challenges](https://wizardsardine.com/blog/the-state-of-bitcoin-inheritance-planning/) when it comes to passing on funds to future generations. Liana addresses this with inheritance policies that activate only after a predefined delay called a "time-lock". For example, a user might control a primary key for daily use, while a recovery key held by a beneficiary is time-locked for future access.
```

Comment:
if you're taking the suggestion to introduce terminology ("time-lock") before using it, that needs to be done before it's done in the first sentence of the article, especially if it's used in the title

Reply:
Good point. Moved the timelock introduction into the introduction